include/mixer.h

```diff
@@ -118,7 +118,7 @@ enum LINE_INDEX : uint8_t {
 	// standard at the host-level, then additional line indexes would go here.
 };
 
-enum class ChannelFeature { Stereo, ReverbSend, ChorusSend };
+enum class ChannelFeature { Stereo, ReverbSend, ChorusSend, AutoSleep };
```

Comment:
Noice! 👍🏻

Reply:
The channel `Features` is such a nice way of toggling stuff on or off! (No need to fatten the constructor with more args)